Installing-and-Configuring/install-os.md

```diff
@@ -0,0 +1,326 @@
+ `/bin/cat`:
+
+`sda5_crypt UUID=XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX /secret.key luks,keyscript=/bin/cat`
+
+(Credit to [http://www.pavelkogan.com/2015/01/25/linux-mint-encryption/](http://www.pavelkogan.com/2015/01/25/linux-mint-encryption/)
+ for this trick).
+
+Installing Qubes
+===
+
+![Heads splash screen]({{ site.baseurl }}/images/Heads_splash_screen.jpg)
+
+Plus in the USB stick with the R4.0 install media into one of the USB3 ports (on
+ the left side of the x230) and boot into USB mode (hit 'u' at the prompt),
+ then boot using this option:
+
+```text
+2. Install Qubes R4.0 [kernel /isolinux/xen.gz console=none]
+```
+
+If that completes with no errors it will launch the Xen hypervisor from the
+ x230's ROM image and start the Qubes installer.  The first few seconds are run
+ with an archaic video mode, so things appear a little weird, but once the dom0
+ kernel initializes the graphics it should look right.
+
+![Qubes partitioning]({{ site.baseurl }}/images/Qubes_partitioning.jpg)
+
+Use default QubesOS partitioning scheme for QubesOS 4.x
+
+![Disk encryption recovery key]({{ site.baseurl }}/images/Disk_encryption_recovery_key.jpg)
+
+The disk encrypt password that you enter here will be used as the
+"recovery password" later.  It should be a long value since you won't
+have to enter it very often; only when upgrading the Heads firmware
+or if there is a need to recover the disk on an external machine.
+You will need it again shortly, so don't lose it yet.
+
+![Signing Qubes binaries in /boot]({{ site.baseurl }}/images/Signing_Qubes_binaries_in__boot.jpg)
+
+Once Qubes has finished installing, you'll need to reboot and select the 'Boot
+ menu' option by hitting 'm'.
+
+Select the first boot option:
+
+```text
+1. Qubes, with Xen hypervisor [...]
+```
+
+Then make this the default boot entry by hitting 'd'.  This will also allow you
+ to seal the disk encryption key.
+
+You will need to input the disk recovery key here (almost for the last time),
+ and this should start the final stage of the Qubes installer.  Under
+ `Configure Qubes` you should select `Create USB qube holding all USB controllers`
+ so that they are protected from outside devices.  This step takes a little
+ while as the templates are configured...
+
+Eventually this will be done and you can click "Finish", then Qubes will
+give you a login screen with your login password.
+
+If you choose to add the disk encryption key to the TPM, you'll need to specify
+ which LUKS volume.  A default Qubes install will work if you leave the
+ 'Encrypted LVM group?' response blank and enter `/dev/sda2` when asked about
+ 'Encrypted devices?'.  For more details see the TPM Disk encryption keys
+ section below. You'll then be asked to enter the disk recovery key as well as
+ the new boot password you'll use to unseal that key.
+
+Once the key sealing process is complete, it will ask you to insert your GPG
+ card then enter your PIN to sign the config.  After this, it will reboot back
```

Comment:
USB Security dongle's User PIN or passphrase. Needs to be the same vocabulary everywhere else everyone gets really really confused.